py_hamt/zarr_encryption_transformers.py

```
@@ -0,0 +1,73 @@
+    ".zarray",
+    # important for coordinate variables, so that we can read bounds
+    "0",
+]
+
+type TransformerFN = Callable[[str, bytes], bytes]
+
+
+def create_zarr_encryption_transformers(
+    encryption_key: bytes,
+    header: None | bytes = None,
+    exclude_vars: list[str] = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)